api/controllers/restaurantController.js

```diff
@@ -1,22 +1,84 @@
 const mockRestaurants = require('../../__mocks__/mockRestaurants');
+const {Restaurant} = require('../models');
 
 const createRestaurant = async (req, res) => {
-  res.sendStatus(200);
+  const {
+    name,
+    description,
```

Comment:
Going to need a property for category that possibly holds **breakfast**, **brunch** **lunch**, **dinner** or **dessert** unless description would already have those keywords.

Reply:
Nevermind I see your reply saying that this will be in a future PR